Eshop.WebApi/Controllers/ProductController.cs

```diff
@@ -7,50 +8,56 @@ namespace Eshop.WebApi.Controllers
     [Route("[controller]")]
     public class ProductController : Controller
     {
-        private static List<Product> Products = new List<Product>
+        private readonly EshopDbContext dbContext;
+
+        public ProductController(EshopDbContext dbContext)
         {
-            new Product(1, "Notebook Acer 16", "Best notebook out there", 399.99m, CategoryController.Categories[0]),
-            new Product(2, "Mouse Razor 123", "Best Mouse", 14.50m, CategoryController.Categories[1]),
-        };
+            this.dbContext = dbContext;
+        }
 
         [HttpGet]
         public List<Product> GetProducts()
         {
-            return Products;
+            return dbContext.Products.ToList();
         }
 
         [HttpGet("{id}")]
         public Product GetProduct(int id)
         {
-            return Products.First(x => x.Id == id);
+            return dbContext.Products.First(x => x.Id == id);
         }
 
         [HttpPost]
         public Product CreateProduct(string title, string description, decimal price, int categoryId)
         {
             var category = CategoryController.Categories.First(x => x.Id == categoryId);
-            var newId = Products.Max(x => x.Id) + 1;
+            var newId = dbContext.Products.Max(x => x.Id) + 1;
             var newProduct = new Product(newId, title, description, price, category);
             
-            Products.Add(newProduct);
+            dbContext.Products.Add(newProduct);
+            dbContext.SaveChanges();
 
             return newProduct;
         }
 
         [HttpDelete("{id}")]
         public void DeleteProduct(int id)
         {
-            var productToBeDeleted = Products.First(x => x.Id == id);
-            Products.Remove(productToBeDeleted);
+            var productToBeDeleted = dbContext.Products.First(x => x.Id == id);
+            dbContext.Products.Remove(productToBeDeleted);
+
```

Comment:
sometimes you have vertical space sometimes not between SaveChanges 
try to keep same look and feel for similar codes, it makes developer experience better.